samza-api/src/main/java/org/apache/samza/runtime/ApplicationRunner.java

```diff
@@ -98,5 +104,5 @@ static ApplicationRunner fromConfig(Config config) {
    * @param streamId  The logical identifier for the stream in Samza.
    * @return          The {@link StreamSpec} instance.
    */
-  StreamSpec streamFromConfig(String streamId);
+  public abstract StreamSpec getStream(String streamId);
```

Comment:
That totally makes sense. I'll put it on my list. We should really revisit all the names before the release, esp. with the architecture diagram so we can fully see the interactions/roles.